README.md

```diff
@@ -81,7 +81,7 @@ shipped with deployinator
 - Run the tailer as a background service (using whatever init flavor you like)
 
 ```sh
-    ./bin/deployinator-tailer.rb &
+    ./deployinator/deployinator-tailer.rb &
```

Comment:
Did you run bundle install --binstubs deployinator? That should put the file in bin. Are you perhaps not using bundler or have a different config for it?